src/lib/wpack.js

```diff
@@ -75,7 +75,26 @@ const run = configs =>
     });
   });
 
+/**
+ * Runs webpack with an array of configurations in series
+ * @param {array} configs Array of webpack configurations
+ * @param {serverless} sls Serverless instance
+ * @returns {Promise} Webpack stats
+ */
+const runSeries = (configs, sls) =>
```

Comment:
You have to call `run` in the reduce to have it run in series, no?

Something like:
```
const runSeries = R.reduce((promise, config) => promise.then(() => run(config)), Promise.resolve());
```